samples/sample.js

```diff
@@ -3,14 +3,15 @@
 const { diag, DiagConsoleLogger, DiagLogLevel } = require('@opentelemetry/api');
 const { MeterProvider } = require('@opentelemetry/metrics');
 const { DynatraceMetricExporter } = require('..');
-const config = require("./config.json");
 
 diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL)
 
 const exporter = new DynatraceMetricExporter({
   prefix: 'sample',
-  url: config.url,
-  APIToken: config.APIToken,
+
+  // URL and API Token are not required if using the OneAgent local endpoint
+  // url: "https://abc12345.live.dynatrace.com/api/v2/metrics/ingest",
+  // APIToken: "API TOKEN HERE"
```

Comment:
```suggestion
  // If no OneAgent is available locally, export directly to the Dynatrace server:
  // url: 'https://myenv123.live.dynatrace.com/api/v2/metrics/ingest',
  // APIToken: '<load API token from secure location such as env or config file>'
```
I'd just use the same wording as in the readme.